### DISCUSSION TITLE
Saluto e informazione generale

### DISCUSSION THREAD

**Discussion by Curadecoco** [indomus/forum#1640] [GENERALE - Discussioni generiche]
Buon giorno a tutti, mi sono da poco iscritto per addentrarmi un po' nel mondo della domotica.
Attualmente ho installato un gateway della Vimar per il controllo delle tende da esterno e di alcune tapparelle oltre che di due luci.
Per quanto il sistema Vimar sia elastico per la creazione di automatismi ho pensato di caricare il tutto su Home Kit di Apple usando attualmente un tablet sempre acceso in casa che funge da gestore.
Volendo aumentare il numero di prodotti da collegare mi rendo conto che il 90% di loro funziona tramite un gateway proprietario, se non ho capito male si può ovviare utilizzando Home Assistant.
Da quanto sto leggendo nelle guide posso caricare in modo più o meno complesso o con addon  sensori o attuatori su  Home Assistant e poi rimbalzare lo stesso su Home di apple per avere alla fine un unico gestore.
Vi chiedo se ho capito bene se così fosse mi attiverei per trovarmi un Raspberry nel instllare la versione più corretta di  Home Assistant.
Grazie dell'aiuto.

**Comment by NdR91** [comment]
Ciao,

Come hai intuito, Home Assistant è un HUB Domotico. La sua peculiarità principale, è proprio quella di avere una serie di integrazioni (ufficiali e non) con una vastissima quantità di dispositivi e brand.

Il mio consiglio spassionato, visto che nonostante tutto parliamo di uno strumento che va studiato e capito, è quello di testarlo prima di spendere anche solo 1€.
Puoi farlo da qualsiasi PC o Mac e senza rinunciare ad alcuna funzione. L'unica "pecca" di questa soluzione, è chiaramente l'inefficienza di tenere acceso un PC h24. Ma ripeto: per eseguire una prima prova, è di gran lunga la scelta migliore.

Sarai sempre in tempo ad acquistare HW dedicato e trasferire la tua configurazione in seguito.
